examples/launcher/src/components/Engine.jsx

```diff
@@ -106,14 +106,20 @@ class Engine extends React.Component {
       }
       return classNames.join(' ');
     }
-    
+
     openMenu(item) {
       const open = this.state.item !== item;
       this.setState({item: open ? item : null}, () => {
         this.postMenuStatus();
       });
     }
-    
+
+    exitExokit() {
```

Comment:
Should this be a different name? `exitStudio()` ?

Reply:
Probably `exitStudio`.